README.md

```diff
@@ -1,4 +1,5 @@
-# ghc-lib [![Hackage version](https://img.shields.io/hackage/v/ghc-lib.svg?label=Hackage)](https://hackage.haskell.org/package/ghc-lib) [![Stackage version](https://www.stackage.org/package/ghc-lib/badge/nightly?label=Stackage)](https://www.stackage.org/package/ghc-lib) [![Build Status](https://dev.azure.com/digitalasset/ghc-lib/_apis/build/status/digital-asset.ghc-lib?branchName=master)](https://dev.azure.com/digitalasset/ghc-lib/_build/latest?definitionId=11&branchName=master)
+# ghc-lib [![Hackage version](https://img.shields.io/hackage/v/ghc-lib.svg?label=Hackage)](https://hackage.haskell.org/package/ghc-lib) [![Stackage version](https://www.stackage.org/package/ghc-lib/badge/nightly?label=Stackage)](https://www.stackage.org/package/ghc-lib) [![ghc-lib-ghc-HEAD-ghc-9.10.1](https://github.com/digital-asset/ghc-lib/actions/workflows/ghc-lib-ghc-HEAD-ghc-9.10.1.yml/badge.svg)](https://github.com/digital-asset/ghc-lib/actions/workflows/ghc-lib-ghc-HEAD-ghc-9.10.1.yml)
```

Comment:
Could we maybe reword the name of the check in the readme? It isn't obvious to an newcomer what the check means

Reply:
Something like `GHC HEAD as/in 9.10.1`?

Reply:
i'm not sure what you want it to read exactly? 

fwiw it's decoded as follows: ghc-flavor = prevailing GHC HEAD commit, build-compiler = ghc-9.10.1.

> Something like GHC HEAD as/in 9.10.1?

did you mean, make it read "GHC HEAD"? anyway, another option is to try to get it to read "Build status" as it did before. would that work better?

Reply:
My point is mostly just that `ghc-lib-ghc-HEAD-ghc-9.10.1` isn't clear to someone unfamiliar with the scheme. Given what you've told me, something like `GHC Head Build` might be good enough, or `Built for GHC Head with 9.10.1`.
Something that has meaning to someone unfamiliar with the project

Reply:
i agree the current tag is only really meaningful to "insiders". i find even "GHC Head Build" and the like also not terribly meaningful when looked at through the lens of a user.

it's fairly common practice to just say "CI" or "Build" for these things, so i advocate "Build" or "Build status". how do you feel about that?

Reply:
I'm happy with that